From 645217bde5b21c88fa07dd5eb00c6dd39d5af72f Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:47 -0400
Subject: [redhat] kabi: Adding symbol blk_sync_queue (block/blk-core.c)

Message-id: <e534283f7fa3d3f0c4d86395d03928fa6f710c5f.1410510773.git.poros@redhat.com>
Patchwork-id: 95973
O-Subject: [PATCH RHEL7.1 047/233] kabi: Adding symbol blk_sync_queue (block/blk-core.c)
Bugzilla: 1062361
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

Bugzilla: 1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361

    Description:
    ============
    blk_sync_queue() is executed prior to executing blk_cleanup_queue().
    This is done during removal of a device from PP control.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/blk_sync_queue
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/blk_sync_queue
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/blk_sync_queue

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/blk_sync_queue b/redhat/kabi/kabi-module/kabi_ppc64/blk_sync_queue
new file mode 100644
index 0000000..9fc35a3
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/blk_sync_queue
@@ -0,0 +1,2 @@
+#1-
+0xfddaf54b blk_sync_queue vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/blk_sync_queue b/redhat/kabi/kabi-module/kabi_s390x/blk_sync_queue
new file mode 100644
index 0000000..3a744b0
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/blk_sync_queue
@@ -0,0 +1,2 @@
+#1-
+0x3784d2ee blk_sync_queue vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/blk_sync_queue b/redhat/kabi/kabi-module/kabi_x86_64/blk_sync_queue
new file mode 100644
index 0000000..82785d4
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/blk_sync_queue
@@ -0,0 +1,2 @@
+#1-
+0xff6fcdb5 blk_sync_queue vmlinux EXPORT_SYMBOL
-- 
1.7.1